pyfstat/mcmc_based_searches.py

```diff
@@ -2158,6 +2086,13 @@ def compute_evidence(self, make_plots=False, write_to_file=None):
 
     @staticmethod
     def read_evidence_file_to_dict(evidence_file_name="Evidences.txt"):
+        """Read evidence file and put it into an OrderedDict
+
+        Parameters
+        ----------
+        evidence_file_name: str
+            Filename to read.
+        """
```

Comment:
missing `Returns`

Reply:
and both missing `Parameters` and `Returns` on `compute_evidence()` above this one